nipype/interfaces/afni/preprocess.py

```diff
@@ -3108,28 +3134,28 @@ class TSmoothInputSpec(AFNICommandInputSpec):
         desc="Sets the data type of the output dataset", argstr="-datum %s"
     )
     lin = traits.Bool(
-        desc="3 point linear filter: 0.15*a + 0.70*b + 0.15*c"
+        desc=r"3 point linear filter: :math:`0.15\,a + 0.70\,b + 0.15\,c`"
         "[This is the default smoother]",
```

Comment:
Lack of space seems to be disrupting RST.

```suggestion
        " [This is the default smoother]",
```